tests/ec2_standalone_tests.sh

```diff
@@ -4,7 +4,18 @@ set -x
 if [ "$RUN_STANDALONE" != "true" ]; then
     exit 0
 fi
+# sometimes the instance profile isn't visible yet to the function.
+# this results in erratic test behavior and tests sometimes failing
 
+n=0
+until [ $n -ge 5 ]
+do
+    aws iam get-instance-profile --instance-profile-name pywren_travis_$BUILD_GUID && break  # substitute your command here
```

Comment:
do you want to pipe this output somewhere (i have no idea what `get-instance-profile` returns). Also this comment can be deleted ?

Reply:
It returns a json blob of the instance profile, but we already know that; that said it's verbose and I will direct it to `/dev/null`